docs/literate/src/files/adding_new_equations_literate/cubic_conservation_law_literate.jl

```diff
@@ -83,18 +83,13 @@ sol = solve(ode, SSPRK43(),
             save_everystep=false, callback=callbacks, maxiters=1e5);
   
 ## Print the timer summary
-summary_callback()
-#src # TODO summary_callback() is printing summary automatically (even with ;)
```

Comment:
I really don't think that we should keep the output. It's too much and not that helpful. I'm open for other solutions than just using a commentary.

Reply:
Yes